src/jit/codegenlegacy.cpp

```diff
@@ -18830,15 +18830,16 @@ regMaskTP CodeGen::genCodeForCall(GenTreeCall* call, bool valUsed)
                                                    (ssize_t)stubAddr);
                             // The stub will write-back to this register, so don't track it
                             regTracker.rsTrackRegTrash(compiler->virtualStubParamInfo->GetReg());
-                            getEmitter()->emitIns_R_R_I(INS_ldr, EA_PTRSIZE, REG_JUMP_THUNK_PARAM,
+                            regNumber indReg = regSet.rsGrabReg(RBM_ALLINT & ~compiler->virtualStubParamInfo->GetRegMask());
+                            getEmitter()->emitIns_R_R_I(INS_ldr, EA_PTRSIZE, indReg,
                                                         compiler->virtualStubParamInfo->GetReg(), 0);
-                            regTracker.rsTrackRegTrash(REG_JUMP_THUNK_PARAM);
```

Comment:
There are other places that use `REG_JUMP_THUNK_PARAM`. Do we really need `REG_JUMP_THUNK_PARAM` - would it make sense to delete it?